### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by shrikantdange1208**
Modified fields as requested by UI team

### CODE REVIEW

File: functions/api/transactions.js [hunk 1]
```diff
@@ -24,7 +23,10 @@ router.get("/", isAdmin, async (request, response, next) => {
         console.log(`fetching document of branch ${doc.id}`);
         const branchSnapshot = await doc.get()
         const branchData = branchSnapshot.data()
-        let subCollectionDocRef = branchCollectionRef.doc(doc.id).collection('transactions');
+        let subCollectionDocRef = branchCollectionRef
+                                    .doc(doc.id)
+                                    .collection('transactions')
```
Comment (ramyarv) [inline functions/api/transactions.js]: can use constant here for transactions

**Review by ramyarv — COMMENTED** [review]
Transaction changes looks ok to me. minor comment given. was there any decision around subgrouping of transactions under each branch while retrieving all transaction cases

File: functions/api/action.js [hunk 2]
```diff
@@ -79,146 +80,63 @@ router.post('/adjustment', async (request, response, next) => {
...
 
-
 module.exports = router;
 
 /**
  * Method to create a new transaction
  * @param {data for the transaction} data
  */
-const createTransaction = async function (data) {
+module.exports.createTransaction = async function (data) {
```
Comment (neerk28) [inline functions/api/action.js]: are you using createTransaction else where outside this file. If not we can remove exports everywhere for createTransaction

Comment (shrikantdange1208) [reply functions/api/action.js]: No. I am not using it outside this file.

File: functions/api/action.js [hunk 3]
```diff
@@ -267,49 +185,15 @@ function getClosingQuantity(operation, initialQuantity, operationalQuantity) {
...
         case constants.ADJUSTMENT:
             return operationalQuantity
     }
 }
 
-/**
- * Trigger to update availableQuantity and isBelowThreshold value after
- * a transaction is completed for a particular branch
- */
-module.exports.updateAvailableQuantityInInventory = functions.firestore
```
Comment (neerk28) [inline functions/api/action.js:280]: why have you removed the trigger?

File: functions/api/pendingRequests.js [hunk 4]
```diff
@@ -12,17 +12,17 @@ const db = admin.firestore();
  */
 router.get("/branches/:id", async (req, res, next) => {
     const branchId = req.params.id
-    const allPendingTransactions = []
+    const allPendingRequests = []
```
Comment (neerk28) [inline functions/api/pendingRequests.js:15]: my bad, missed it. Thanks for correcting it :)

File: functions/api/action.js (hunk 2 above)
Comment (shrikantdange1208) [reply functions/api/action.js]: Done

**Review by neerk28 — APPROVED** [review]
LGTM, Merging the changes.

**Pull request merged**